tests/test_hgexports.py

```diff
@@ -604,7 +608,7 @@ def test_check_wpt_sync_irrelevant_user(mocked_repo_config):
         GIT_DIFF_FILENAME_TEMPLATE.format(filename="somefile.txt")
     )
     diff_assessor = DiffAssessor(
-        author="sheehan@mozilla.com",
+        email="sheehan@mozilla.com",
```

Comment:
What was this change for? Maybe just add something to the commit message.

Reply:
Added extra context to commit message.